app/src/main/java/de/mathfactory/mooltifill/SettingsActivity.kt

```diff
@@ -201,6 +205,40 @@ class SettingsActivity : AppCompatActivity() {
 //        }
     }
 
+    private fun scheduleBlutoothStateObserver()
+    {
+        val filter = IntentFilter()
+        filter.addAction(BluetoothDevice.ACTION_ACL_CONNECTED)
+        registerReceiver(deviceStatesReceiver, filter)
+    }
+
+    private val deviceStatesReceiver: BroadcastReceiver = object : BroadcastReceiver() {
```

Comment:
Suggestion when using the previously mentioned method of registering: Implement the BroadcastReceiver in a new file like the already existing BootReceiver.kt

Reply:
Done in the local commit.